ui/src/app/metadata/detail/abstract-metadata-detail.component.ts

```diff
@@ -75,13 +76,13 @@ export abstract class AbstractMetadataDetailComponent extends QuickLinksHelper i
   }
 
   get showSkillsTable(): boolean {
-    return !this.skillTableControl.emptyResults;
+    return this.skillTableControl.skills.length > 0
   }
 
-  // get skillsCountLabel(): string {
-  //     const rsdLabel = (this.metadata?.skillCount == 1) ? "RSD" : "RSDs"
-  //     return `${this.skillTableControl.totalCount} ${rsdLabel} with this category based on`
-  // }
+  get skillsCountLabel(): string {
+    const rsdLabel = (this.skillTableControl.size == 1) ? "RSD" : "RSDs"
+    return `${this.skillTableControl.totalCount} ${rsdLabel} with this category based on`
```

Comment:
Update the word "category" to use the correct metadata type when displaying text to the user:
"... `${rsdLabel}` with this `${metadataType}` based on..."